docs/source/install.rst

```diff
@@ -0,0 +1,75 @@
+Installing with ``pip``
+=======================
+
+Installing Zipline via ``pip`` is slightly more involved than the average
+Python package.
+
+There are two reasons for the additional complexity:
+
+1. Zipline ships several C extensions that require access to the CPython C API.
+   In order to build the C extensions, ``pip`` needs access to the CPython
+   header files for your Python installation.
+
+2. Zipline depends on `numpy <http://www.numpy.org/>`_, the core library for
+   numerical array computing in Python.  Numpy depends on having the `LAPACK
+   <http://www.netlib.org/lapack>`_ linear algebra routines available.
+
+Because LAPACK and the CPython headers are non-Python dependencies, the correct
+way to install them varies from platform to platform.  If you'd rather use a
+single tool to install Python and non-Python dependencies, or if you're already
+using `Anaconda <http://continuum.io/downloads>`_ as your Python distribution,
+you can skip to the :ref:`Installing with Conda <conda>` section.
+
+Linux
```

Comment:
@llllllllll I made the section heading here `GNU/Linux`, but I left the in-line references as just `Linux` in the interest of readability.